guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: guides
+title: Creating your personal mixed skin
+description: "This guide teaches you how to create your own personal mixed skin."
+current: Guides | Mixing a skin

```

Comment:
no longer used, use this for the title instead, probably should also be named "Mixing Skins" instead